Wikipedia/Code/BasicLogger.swift

```diff
@@ -0,0 +1,7 @@
+@objc class BasicLogger: NSObject, DDLogger {
+    func log(message logMessage: DDLogMessage) {
+        print(logFormatter?.format(message: logMessage) ?? logMessage.message)
```

Comment:
Couldn't find a way to knock off the gross start of lines without doing it this way. Sample project also included that NSLog-ish start of line, until I did something like this.